manage.sh

```diff
@@ -57,22 +57,27 @@ function install_conda {
     esac
 
     if conda --version ; then
-        echo Conda already installed. Skipping conda installation.
+	if  [ `(conda --version | cut -d' ' -f2)` == "4.8.2" ]; then
+            echo Conda already installed.
+	else
+	    conda activate base
+	    conda install conda=4.8.2
```

Comment:
`conda deactivate` after update?

Reply:
Another thing that is bothering me is that, as I understand it, the update affects all environments and therefore it has side effects for other environments we may have (I have a few, at least). Is this the case? If so maybe we shouldn't force the conda version. If not then this should be done just for the IC environment.

Reply:
I dont think there should be any problems with keeping old environments with updated conda version, but maybe is safer to just print a message saying that the conda is outdated and one can update it with a commant